gubernator/main.py

```diff
@@ -176,6 +191,31 @@ def build_details(build_dir):
                          build_log.count('\n'))
     return started, finished, failures, build_log
 
+def parse_kubelet(pod, junit, build_dir):
+    junit_file = "junit_" + junit + ".xml"
```

Comment:
Pass the full filename in, instead of the number. Some tests don't have 2-digit numbered junit files.